Dockerfile.ci.test

```diff
@@ -0,0 +1,16 @@
+# This image is used for testing OpenShift Console plugin template capabilities.
+
+FROM registry.access.redhat.com/ubi8/nodejs-16:latest AS build
```

Comment:
Can we simply make these changes to the normal Dockerfile and not need a separate Dockerfile.ci.test?

Reply:
> Can we simply make these changes to the normal Dockerfile and not need a separate Dockerfile.ci.test?

@spadgett Updated.